packages/native-pack-tool/src/base/default.ts

```diff
@@ -551,6 +552,10 @@ export class CocosParams<T> {
     public platform: string;
     public platformName: string;
     public executableName: string;
+    /**
+     * batch file directory of native ide
+     */
+    public nativeIdeDir: string;
```

Comment:
Is nativeIdeDir the only params needed, and do you need to consider more params?

Reply:
Only one parameter is needed for Android.

Reply:
I mean is that this param is set in base, and if there are other params that need to be used when the project is opened on other platforms, are they all put in an object instead of declaring a param